security/requests.py

```diff
@@ -8,8 +8,9 @@
 
 class SecuritySession(Session):
 
-    def __init__(self, slug=None, related_objects=None):
+    def __init__(self, logger_class=OutputRequestLogger, slug=None, related_objects=None):
```

Comment:
another thing. When You add an option argument add it to the end of the argument list, please. Not at the beginning.